qml/components-Sailfish/IconButton.qml

```diff
@@ -0,0 +1,30 @@
+import Sailfish.Silica 1.0
+import "." as BVApp
+
+IconButton {
+    function iconBy(type) {
```

Comment:
My idea was that it actually returns the icon type that is used on the corresponding platform: A string on v-play and a (scaled) icon on Sailfish, which both can be directly assigned to the icon object of the ```IconButton```. But I just saw that we then need to export the ```pressed``` property as a parameter of this function, as well, right? Does v-play have sth. like this? I assume yes, and I assume it is named differently. ;)